app/main.py

```diff
@@ -1 +1,19 @@
-# write your code here
+def copy_file(command: str) -> None:
+    linux_command, source_name, new_file = command.split(" ")
+    if (
+            linux_command != "cp" or not (
+            source_name.endswith(".txt")
+            or new_file.endswith(".txt")
+            )):
+
+        raise ValueError("Check your command and try again")
+    if source_name == new_file:
+        raise ValueError("Names should not have the same name")
```

Comment:
actually in task was written: does nothing. So, probably, it should do nothing, not Raise an error, even though it's intuitive.